### DISCUSSION TITLE
Let's point version of all packages in every repository

### DISCUSSION THREAD

**Discussion by veebull** [Cyfrin/foundry-full-course-cu#1025] [Ideas]
Now I learn UUPS and now library of proxies of OZ is 5 but in course necessary is 4.8.3. And that's why major upgrade breaks test in lesson. It would be nice to add specific version for all libraries

**Comment by cromewar** [comment]
They are pointed on [chronological-updates.md](https://github.com/Cyfrin/foundry-full-course-f23/blob/main/chronological-updates.md), but we should also put them on the course index, thanks for the recommendation @armbull!
